### PR TITLE
Add regression test to make sure we can load old containers with volumes.

### PR DESCRIPTION
**Pull request by calavera**
Signed-off-by: David Calavera david.calavera@gmail.com

### CODE REVIEW

**Comment by cpuguy83** [comment]
Janky is sad.

**Comment by calavera** [comment]
yeah, fixing all that

**Comment by calavera** [comment]
Janky is happy again :smile:

File: daemon/daemon_test.go [hunk 1]
```diff
@@ -113,3 +119,88 @@ func TestGet(t *testing.T) {
...
+	err = daemon.verifyOldVolumesInfo(c)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(c.MountPoints) != 1 {
+		t.Fatalf("Expected 1 volume mounted, was 0\n")
+	}
+
+	m := c.MountPoints["/tmp/vol1"]
```
Comment (tiborvass) [inline daemon/daemon_test.go]: `/vol1` instead, to avoid any side effect with `/tmp`

**Comment by calavera** [comment]
@tiborvass fixed

**Comment by tiborvass** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**